README.md

```diff
@@ -6,28 +6,20 @@ A web frontend for integrating a google calendar with jitsi meetings. The intent
 
 ### Prerequisites
 
-The application must be built first in order to run. A pre-requisite to building is having node and npm installed; specific versions that are known to work should be listed in package.json. Another prerequisite is filling out `src/config/index.js`. The file is left out intentionally so people can add their own configurations. Here is a sample:
-
-```
-export CLIENT_ID = 'id-here';
-export XMPP_CONFIG = {
-    bosh: 'bosh-url',
-    hosts: {
-        domain: 'host-url',
-        muc: 'muc-url',
-        focus: 'focus-url'
-    }
-}
-```
-
-The XMPP_CONFIG is what is expected by `lib-jitsi-meet` so please view that repository for more information.
+The application must be built first in order to run. A pre-requisite to building is having node and npm installed; specific versions that are known to work should be listed in package.json. Another prerequisite is modifying `src/config/index.js`. The file is filled with defaults but should be customized as needed.
+
+The XMPP_CONFIG in config is what is expected by `lib-jitsi-meet` so please view that repository for more information.
 
 ### Installing
 
 To build the frontend, run `npm install` to download dependencies and `npm run build-prod` to build the javascript bundle. Next serve `index.html` from a server, such as with `python -m SimpleHTTPServer`.
 
 If developing, running `npm run watch-dev` to build the javascript will run webpack in watch mode to automatically rebundle on changes and also enable debug features.
 
+### Special note about ultrasound support
+
+The app currently uses Quiet.js for ultrasound. The spot client can emit an ultra sound message for remote clients to be notified of which remote control url to go to. The Quiet.js files are currently in the top level of the project and should not be modified directly as they are third-party files.
```

Comment:
`remote control url` is a little confusing i think.  is it right that this is referring to "the url you go to on any web client to remotely control the spot in order to join a call, etc"?